### PR TITLE
Jose/fix get values

### PR DESCRIPTION
**Pull request by jotathebest**

### CODE REVIEW

File: src/Ubidots.cpp [hunk 1]
```diff
@@ -26,60 +26,40 @@ Modified by Jose Garcia for Ubidots Inc
...
-/** 
- * This function is to set UDP or TCP method
- * @arg method is the method that you want to use
- * @return true uppon succes
- */
-void Ubidots::setMethod(uint8_t method) {
-    if (method >= 0 && method <= 2) {
-        _method = method;
-    }
+    strcpy(_dsName, str.c_str());
```
Comment (woakas) [inline src/Ubidots.cpp]: I think that is not necessary to put the ParticleID in the dsName.

File: src/Ubidots.cpp [hunk 2]
```diff
@@ -387,22 +478,70 @@ void Ubidots::add(char *variable_id, double value) {
...
+/**
+ * This function is to set the name of your device to visualize,
+ * if you don't call this method the name by default will be 'Particle'
+ * @arg deviceName is the name to display in Ubidots, avoid to use special
+ * characters or blank spaces
+ * @return true uppon succes
+ */
+
+void Ubidots::setDeviceName(char* deviceName) {
+    Serial.print("Warning, this function is deprecated,");
```
Comment (woakas) [inline src/Ubidots.cpp]: This function is new, it should not be deprecated.

File: src/Ubidots.cpp [hunk 3]
```diff
@@ -387,22 +478,70 @@ void Ubidots::add(char *variable_id, double value) {
...
+ * This function is to set your device label, the device
+ * label is the unique device identifier in Ubidots.
+ * if you don't call this method the name by default will be the device ID
+ * @arg deviceLabel is the device label, avoid to use special
+ * characters or blank spaces
+ * @return true uppon succes
+ */
+
+void Ubidots::setDeviceLabel(char* deviceLabel) {
+    Serial.print("Warning, this function is deprecated,");
```
Comment (woakas) [inline src/Ubidots.cpp]: New Function, it should not be deprecated

File: src/Ubidots.cpp (hunk 1 above)
Comment (jotathebest) [inline src/Ubidots.cpp]: it should be the label not the name the one that should have the device ID. Fixing for next commit.

File: src/Ubidots.cpp (hunk 2 above)
Comment (jotathebest) [inline src/Ubidots.cpp]: Fixing for next commit.

**Pull request merged**